diff --git a/node_modules/@huggingface/transformers/dist/transformers.node.mjs b/node_modules/@huggingface/transformers/dist/transformers.node.mjs
index 16d7f7b..ecff807 100644
--- a/node_modules/@huggingface/transformers/dist/transformers.node.mjs
+++ b/node_modules/@huggingface/transformers/dist/transformers.node.mjs
@@ -32008,6 +32008,8 @@ async function getModelFile(path_or_repo_id, filename, fatal = true, options = {
             && typeof Response !== 'undefined' // 2. `Response` is defined (i.e., we are in a browser-like environment)
             && response instanceof Response    // 3. result is a `Response` object (i.e., not a `FileResponse`)
             && response.status === 200         // 4. request was successful (status code 200)
+    }else{
+      cacheKey=localPath
     }
 
     // Start downloading
@@ -32100,7 +32102,7 @@ async function getModelFile(path_or_repo_id, filename, fatal = true, options = {
     // Otherwise, return the cached response (most likely a `FileResponse`).
     // NOTE: A custom cache may return a Response, or a string (file path)
     const cachedResponse = await cache?.match(cacheKey);
-    if (cachedResponse instanceof FileResponse) {
+    if (return_path) {
         return cachedResponse.filePath;
     } else if (cachedResponse instanceof Response) {
         return new Uint8Array(await cachedResponse.arrayBuffer());
